task6_Shai_Axelrod/Task06/src/main.cpp

```diff
@@ -0,0 +1,194 @@
+#include <iostream>
+#include <string>
+#include <algorithm>
+#include "DataBase.h"
+
+/**
+ * @brief Displays the menu and retrieves user inputs for city, radius, and norm.
+ * @param city A reference to the string variable to store the selected city.
+ * @param radius A reference to the string variable to store the wanted radius.
+ * @param norm A reference to the string variable to store the wanted norm.
+ * @param db The database.
+ */
+void Menu(std::string& city, std::string& radius, std::string& norm, DataBase& db);
```

Comment:
it's better if you passed these function to another class to handle the logic instead of the main